modules/generators/content/password_file/templates/password_file.md.erb

```diff
@@ -0,0 +1,14 @@
+# Password list
+
+<%
+    website_count = 1
+    website_sample.each { |wsite|
+%>
+## <%= wsite %>
+### Username: <%= username_sample[website_count] %>
+### Password: <%= pass_list[rand(1..10)] %>
```

Comment:
Choosing the passwords at random could mean that of all the passwords fed in to the module they might not all actually be used. Would be better to iterate through the list of passwords.